doc/rbd/iscsi-target-cli.rst

```diff
@@ -0,0 +1,167 @@
+=============================================================
+Configuring the iSCSI Target using the Command Line Interface
+=============================================================
+
+The Ceph iSCSI gateway is the iSCSI target node and also a Ceph client
+node. The Ceph iSCSI gateway can be a standalone node or be colocated on
+a Ceph Object Store Disk (OSD) node. Completing the following steps will
+install, and configure the Ceph iSCSI gateway for basic operation.
+
+**Requirements:**
+
+-  A running Red Hat Ceph Storage 2.3 cluster
+
+-  The following packages must be installed from a Red Hat Ceph Storage
+   repository:
+
```

Comment:
@dillaman : Is it safe to say CentOS 7.4 instead of RHEL 7.4?  Do we want to make any reference to Red Hat upstream?  I'm just curious.

Also somewhat related, what community online repository should we be pointing users to so they can install the RPM packages?  I'm assuming we will have them just `wget <packages>`, then they can use `yum`, but we'll need to know "where" to get them.